### DISCUSSION TITLE
Location of Files

### DISCUSSION THREAD

**Discussion by Stoonman** [abraunegg/onedrive#2316] [Q&A] [answered]
Hi.  This is my first time trying to do something like this, so forgive me if I don't know what I'm doing and/or saying.  I looked for an answer, but I couldn't find anything.  It could be that I just don't know how to search properly, though.  

I believe that I've got it installed correctly because I tried the dry run command it all seemed correct.  But, now I need to change how it is set up (configured?).  It's linked to the solid state drive, but I need it to use the larger spinning drive.  How do I do that?  Any advice is appreciated.  Thanks in advance.  

In case this is important:  I'm using Linux Mint 21.1 Cinnamon; the cinnamon version is 5.6.7

**Comment by abraunegg** [comment, marked as answer]
@Stoonman 
The location of your OneDrive files is by default stored in your users home directory - `~/OneDrive`

This is governed by the 'sync_dir' configuration option as per https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#sync_dir

In your scenario you have 2 options available to you:
* Configure 'sync_dir' explicitly to point at your desired location
* Utilise a symbolic link to point `~/OneDrive` --> `/your/desired/destination/location`


Please have a read of the usage document (https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md) to assist you further.
